gradlew

```diff
@@ -246,4 +246,5 @@ eval "set -- $(
         tr '\n' ' '
     )" '"$@"'
 
-exec "$JAVACMD" "$@"
+# Re-generate locks on every task
+exec "$JAVACMD" "$@" --write-locks
```

Comment:
Is this something we _want_ on every task or should it be done on purpose when changing a dependency?

Reply:
I borrowed this trick from another project. The problem is that ppl will always forget to re-generate the lock